src/cli/cli.cpp

```diff
@@ -1427,6 +1446,32 @@ template <> otError Interpreter::Process<Cmd("channel")>(Arg aArgs[])
             SuccessOrExit(error = aArgs[2].ParseAsBool(enable));
             otChannelManagerSetAutoChannelSelectionEnabled(GetInstancePtr(), enable);
         }
+#endif // OPENTHREAD_FTD
+#if OPENTHREAD_CONFIG_CHANNEL_MANAGER_CSL_CHANNEL_SELECT_ENABLE
+        /**
+         * @cli channel manager autocsl
+         * @code
+         * channel manager autocsl 1
+         * channel manager autocsl 1
+         * Done
+         * @endcode
+         * @cparam channel manager autocsl @ca{enable}
+         * `1` is a boolean to `enable`.
+         * @par
+         * `OPENTHREAD_CONFIG_CHANNEL_MANAGER_ENABLE` or `OPENTHREAD_CONFIG_CHANNEL_MANAGER_CSL_CHANNEL_SELECT_ENABLE`,
+         * and `OPENTHREAD_CONFIG_CHANNEL_MONITOR_ENABLE` are required.
+         * @par api_copy
+         * #otChannelManagerSetAutoChannelSelectionEnabled
```

Comment:
```suggestion
         * @par
         * Enables or disables the auto channel selection functionality for a CSL channel.
         * @sa otChannelManagerSetAutoCslChannelSelectionEnabled
```

Probably should be otChannelManagerSetAutoCslChannelSelectionEnabled (since that's what's used below), but that API description contains details beyond the use of the CLI command, so I'd recommend using the suggestion to just reference the API and have a shorter description.